AutomatedTesting/Gem/PythonTests/assetpipeline/asset_processor_tests/assets/C1564073/test_prefab.prefab

```diff
@@ -0,0 +1 @@
+WWWWWWWWWW
```

Comment:
I'm a bit confused what this asset is for.  There's one below that's labelled corrupted, with the same contents.  I also saw a test which was replacing {} with WWWWWWWW in a prefab.  Just wondering what this one is used for

Reply:
Yeah there are repeated assets, I can go back and consolidate them.

Reply:
Not a big deal really, it makes sense these tests have their own set of assets.  I guess I'm just more surprised that we have several different tests that are working with an invalid asset.